# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../luatar/use-pkgconfig.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2005-09-14 02:26:44.000000000 +0200
+++ ./Makefile	2006-10-09 13:23:44.000000000 +0200
@@ -23,8 +23,8 @@
 INSTALL_DATA = $(INSTALL) -m 644
 
 LUA=lua
-LUA_CDIR = $(shell lua-config --pkglibdir)
-LUA_LDIR = $(shell lua-config --pkgdatadir)
+LUA_CDIR = $(shell pkg-config --variable INSTALL_CMOD lua)
+LUA_LDIR = $(shell pkg-config --variable INSTALL_LMOD lua)
 
 ifeq ($(DEBUG),0)
 else
